.devcontainer/Dockerfile

```diff
@@ -1,5 +1,8 @@
-# [Choice] bionic (18.04), focal (20.04)
-ARG VARIANT="focal"
+# usage:
```

Comment:
We can use `setup-cpp` for the Dockerfile as well. It could be a separate PR though

Reply:
I am not sure what you mean?

Reply:
We can use this docker file:
https://github.com/aminya/setup-cpp/blob/master/dev/docker/setup-cpp-ubuntu.dockerfile

Reply:
done, but there are small issues:
```
root@753a45f4b678:~/cpp_vcpkg_project# pip3 check
pygobject 3.42.1 requires pycairo, which is not installed.
nala 0.11.1 has requirement rich<13.0.0,>=12.1.0, but you have rich 11.2.0.
root@753a45f4b678:~/cpp_vcpkg_project# 
```

Reply:
and `CMake v3.26` dosn't support `cmake workflow presets`!